Update evmtraining2 20230511 (#167)
godwokenrises/godwoken-doc: docs/evm_training/evmTask2.md

## Task Instructions

In this task, you will 
<div class="task-wrapper">

> - Take a very basic smart contract written in Ethereum's Solidity.

Reviewer: ```suggestion
- Take a very basic smart contract written in [Solidity](https://soliditylang.org/).
```

Author: done

- Compile it to EVM bytecode.

> - Deploy it to Godwoken on Layer 2.

Reviewer: ```suggestion
- Deploy it to [Godwoken Testnet](https://v1.testnet.gwscan.com/)
```

Author: done

### Prerequisites

Before you begin this task, you must complete the [previous task](evm_training/evmTask1.md) to create a Godwoken account on the EVM Layer 2 Testnet. You will also need the private key and Testnet address from the previous task in order to proceed. If you have not completed it, please do so now.
<div class="task-wrapper">

- [Godwoken Account on EVM layer 2 Test](evm_training/evmTask1.md)
- [Private Key](https://docs.godwoken.io/evm_training/evmTask1#1-go-to-godwoken-bridge)

> - [Testnet address](/evm_training/evmTask1#1-go-to-godwoken-bridge)

Reviewer: Testnet address is not used in this page?

}
```

### 3. Deploy a Smart Contract

> Hardhat will handle the process of compilation and deployment of the smart contract.

Reviewer: ```suggestion
[Hardhat](https://hardhat.org/) will handle the process of compilation and deployment of the smart contract.
```
